src/main/java/com.gigssandbox/response/ResponseStore.java

```diff
@@ -12,12 +13,11 @@ public ResponseStore() {
         try {
             properties.load(new FileInputStream("src/main/resources/response.properties"));
         } catch (IOException e) {
-            e.printStackTrace();
-            System.exit(1);
+            throw new LoadingResponsePropertiesException();
         }
     }
 
-    public String stored(String propertyName) {
+    public String getStored(String propertyName) {
```

Comment:
public String loadText(String propertyName)

Reply:
can I simply name it load(String propertyName)?

Reply:
you may but its much clear that this public method returns you raw text